pkg/apis/batch/types.go

```diff
@@ -22,6 +22,10 @@ import (
 	api "k8s.io/kubernetes/pkg/apis/core"
 )
 
+// Unprefixed labels are reserved for end-users so we will add a batch.kubernetes.io to designate these labels as official Kubernetes labels.
+// See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions
+const LabelPrefix = "batch.kubernetes.io/"
```

Comment:
This doesn't need to be exported.
```suggestion
const labelPrefix = "batch.kubernetes.io/"
```